iceoryx_hoofs/include/iceoryx_hoofs/cxx/poor_mans_heap.hpp

```diff
@@ -31,6 +31,9 @@ class PoorMansHeapType
 {
 };
 
+constexpr uint64_t POOR_MANS_HEAP_DEFAULT_ALIGNMENT{8};
+
+// AXIVION Construct Ruleset-A12.1.1 : it is guaranteed that the memory is initialized on access
```

Comment:
You mean the AUTOSAR rule, correct?

```suggestion
// AXIVION Construct AutosarC++19_03-A12.1.1 : it is guaranteed that the memory is initialized on access
```

Reply:
:grimacing: correct